CHANGELOG.md

```diff
@@ -19,6 +20,7 @@
 ### Fixed
 - Fixed bug in `DoctrineResultCacheWarmerCommand` where lack of order by clause caused wrong pagination with PostgreSQL.
 - Fixed `OAuthSignatureValidationSubscriber` to read LTI credentials from configuration instead of database.
+- Fixed Code Style issues in `InfrastructureIngesterTest` and `IngesterCommandTest` files
```

Comment:
Why did you add this under 1.8.1? I suppose it was by mistake

Reply:
Sorry. My bad. Fixed :)